public/js/orders/shopping-cart.js

```diff
@@ -12,19 +16,19 @@ class ShoppingCart {
         this.products = [];
     }
 
-    isProductInShoppingCart(id, size) {
+    isProductInShoppingCart(id, sizeId) {
         for (let i = 0; i < this.products.length; i++) {
-            if (this.products[i].id == id && this.products[i].size == size) {
+            if (this.products[i].id == id && this.products[i].sizeId == sizeId) {
```

Comment:
```suggestion
            if (this.products[i].id === id && this.products[i].sizeId === sizeId) {
```

Reply:
Tijdens het vergelijken van php en js is de ene een string en de ander een int, een cast zou hier wel kunnen.